common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -276,19 +276,15 @@ protected Runnable takeTask() {
     }
 
     private boolean fetchFromScheduledTaskQueue() {
-        if (hasScheduledTasks()) {
-            long nanoTime = AbstractScheduledEventExecutor.nanoTime();
-            for (;;) {
-                Runnable scheduledTask = pollScheduledTask(nanoTime);
-                if (scheduledTask == null) {
-                    break;
-                }
-                if (!taskQueue.offer(scheduledTask)) {
-                    // No space left in the task queue add it back to the scheduledTaskQueue so we pick it up again.
-                    scheduledTaskQueue().add((ScheduledFutureTask<?>) scheduledTask);
-                    return false;
-                }
+        long nanoTime = AbstractScheduledEventExecutor.nanoTime();
+        Runnable scheduledTask  = pollScheduledTask(nanoTime);
+        while (scheduledTask != null) {
+            if (!taskQueue.offer(scheduledTask)) {
+                // No space left in the task queue add it back to the scheduledTaskQueue so we pick it up again.
+                scheduledTaskQueue().add((ScheduledFutureTask<?>) scheduledTask);
+                return false;
             }
+            scheduledTask  = pollScheduledTask(nanoTime);
```

Comment:
Doh! Did not spot it.... Will cleanup

> Am 01.07.2016 um 17:24 schrieb Scott Mitchell <notifications@github.com>:
> 
> In common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java:
> 
> >              }
> > +            scheduledTask  = pollScheduledTask(nanoTime);
> cleanup: remove extra space before =.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or mute the thread.
>